roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -22,6 +27,26 @@
     HAVE_NWB = False
 
 
+def dict_recursive_update(metadata_base, metadata_input):
+    return_dict = deepcopy(metadata_base)
+    for base_key, base_val in return_dict.items():
+        if metadata_input.get(base_key):
```

Comment:
I think `if metadata_input.get(base_key)` will always evaluate to `True` given the line above

Reply:
metadata input may not have the same keys as the metadata_base since it is what the user inputs